src/EliminateBoolVectors.cpp

```diff
@@ -136,6 +136,8 @@ class EliminateBoolVectors : public IRMutator {
         }
     }
 
+    // FIXME: what about Reinterpret?
```

Comment:
@abadams what about this one?
I do believe this is a correct translation, but is that the correct handling in reality?